BUILDING.md

```diff
@@ -1,21 +1,17 @@
-# Build-System CLI
+# Getting started / building from source
 
-## Non-interactive
-```
-python build.py -h, --help
+1. clone the source code from the [J2V8 GitHub repository](https://github.com/eclipsesource/J2V8)
```

Comment:
I put this point already into my backlog/roadmap for future features ... but I don't want to blow up this PR even further. Let's take one leap at a time, so to speak 😆